### PR TITLE
Set up project - related #12

### PR DESCRIPTION
**Pull request by sara219**
- [x] create `create-react-app` basic setup    
using ` npx create-react-app my-app`

- [x] install the following modules
    - react-router-dom
    - yup
    - prop-types
    - antd

using ` npm i react-router-dom` + ` npm i yup prop-types antd`

- [x] setup Travis  
- [x] Deploy on Heroku 
- [x] Eslint 
using `npm install eslint -D`

### CODE REVIEW

**Review by someyaaltous — APPROVED** [review]

**Review by hshahwan — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './App.css';
```
Comment (95Rawan) [inline src/App.js]: No need for this line

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #09d3ac;
+}
```
Comment (95Rawan) [inline src/App.css]: Delete this file we don't need it, we'll put our own style

File: src/App.js (hunk 1 above)
Comment (sara219) [reply src/App.js]: @95Rawan  I delete it

File: src/App.css (hunk 2 above)
Comment (sara219) [reply src/App.css]: @95Rawan  Done!

**Review by 95Rawan — APPROVED** [review]

**Pull request merged**